Meadowlark-js/packages/meadowlark-core/src/middleware/AuthorizationMiddleware.ts

```diff
@@ -31,6 +31,7 @@ export async function authorize({ frontendRequest, frontendResponse }: Middlewar
   frontendRequest.middleware.security = {
     authorizationStrategy: jwtStatus.authorizationStrategy as AuthorizationStrategy,
     clientId: jwtStatus.clientId ?? 'UNKNOWN',
+    validateResources: !jwtStatus.roles.includes('assessment'),
```

Comment:
Since this isn't strictly a "security" thing, what do you think about moving the `validateResources` property to the `middleware` object?

Reply:
Yep, can do.